main/src/ca/uwaterloo/flix/language/phase/typer/ConstraintSolver2.scala

```diff
@@ -125,6 +125,15 @@ object ConstraintSolver2 {
     }
   }
 
+  /**
+    * Returns `true` if `tpe1` and `tpe2` are equivalent types.
+    */
+  def equivalent(tpe1: Type, tpe2: Type)(implicit eqenv: EqualityEnv, flix: Flix): Boolean = {
```

Comment:
Nitpick: I prefer functions that are verbs.

Reply:
-> isEquivalent